docker/podman-docker.sh

```diff
@@ -0,0 +1,8 @@
+# DOCKER_HOST initialization
+
+[ -z "$DOCKER_HOST" ] || return
+if [ "$EUID" -eq 0 ]; then
```

Comment:
This variable doesn't seem to be standardize (dash doesn't define $EUID, if anything other than bash matters, so it errors out if $DOCKER_HOST isn't set).

Reply:
Same as above, this is only trying to prevent the case where XDG_RUNTIME_DIR is set for a root user.